packages/hooks/src/useCreateOffer.ts

```diff
@@ -9,43 +9,60 @@ import { OfferType, TransactionFragment } from './graphql'
 import { convertTx } from './utils/transaction'
 
 gql`
-  mutation CreateOffer(
-    $input: OfferInputBis!
+  query FetchAssetAndCurrency(
```

Comment:
Let's rename that into fetch approval or something related to the approval

Reply:
This query is going to be removed with https://github.com/liteflow-labs/liteflow-js/pull/117.
The two PRs 119 and 117 modify the same queries. A bit hard to follow...
Will be better when new version of backend and api-graphql is released an we can merge everything.

Reply:
let's not touch it then